src/Atoms/HomeElements/HomeContent.test.js

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react';
+
+import HomeContent from './HomeContent';
+
+const data = {
+  item: {
+    snippet: {
+      thumbnails: {
+        high: {
+          url: 'https://yt3.ggpht.com/ytc/AAUvwnighSReQlmHl_S_vSfvnWBAG5Cw4A0YxtE0tm5OpQ=s800-c-k-c0xffffffff-no-rj-mo',
+        },
+      },
+    },
+  },
+};
+
+describe('Home Content', () => {
+  test('should contains a button', () => {
+    const result = render(HomeContent(data));

```

Comment:
You should be able to do: `render<HomeContent data={data}>` or the name of the prop you are expecting in the HomeContent component

Reply:
I think the problem is that all your component files are `.js`. If you changed to `.jsx` and remove all the `React.createElement` and follow the React Best practices, you would be able to improve also your tests. See: https://www.codementor.io/@faizanhaider/react-components-best-practices-9xcbq5uwe